grammers-tl-parser/src/tl/definition.rs

```diff
@@ -146,55 +143,49 @@ impl FromStr for Definition {
                     None
                 }
 
-                // If the parameter is a flag definition save both
-                // the definition and the parameter.
-                Ok(Parameter {
```

Comment:
I can never remember the syntax order but maybe we could've used `@ p`.

This is fine too though, just more nesting.

Reply:
I tried `p @ …` first; I couldn't resolve the resulting mutable reference issues.